CSS/components/card.css

```diff
@@ -0,0 +1,34 @@
+.flex-container {
```

Comment:
I would expect a more meaningful class name here especially since you already named the CSS-file 'card'. Maybe a name like 'card' is more descriptive and accurate?!